### ISSUE TITLE
Review and update (where necessary) kit/software documentation

### ISSUE THREAD

**Issue by andybusse** [srobo/core-team-minutes#12]
See also #34 (Python 3) and #35 (Odroid OS).

**Comment by PeterJCLaw** [comment]
This depends on knowing what kit we're going to ship (blocked by #11) and on what tooling we're going to provide the teams (#13).

However we should still be able to progress the software API side of things. We know that we want to move to Python 3 if at all possible, so we should start looking at whether the SR software stack can be moved to using Python 3.

**Comment by PeterJCLaw** [comment]
Closing in favour of https://github.com/srobo/docs/issues/60, especially since we're not actually changing anything about the kit.
